.circleci/config.yml

```diff
@@ -319,6 +321,14 @@ commands:
           # Stop running Puppeteer end-to-end test if code in relevant directories is unchanged.
           dir_names: "ui e2e .circleci"
 
+  puppeteer-access-test-user-setup:
```

Comment:
Making `puppeteer-access-test-user-setup` as a CI command instead a CI job. Easier to call in other jobs.